cirq/aqt/aqt_sampler.py

```diff
@@ -0,0 +1,226 @@
+import json
+import time
+import uuid
+from typing import Iterable, List, Union
+
+import numpy as np
+from requests import put  # TODO: Specify as a dependency
```

Comment:
This TODO has to be resolved before we can merge. You would add this requirement into the `requirements.txt` file.

Reply:
I added `requests` to `requirments.txt`. I can try to use `urllib` from the standard library if you do not want to have this dependency. Will result in messier code though.

Reply:
Nevermind. requests was already in the requirements